CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.3.0...HEAD
 ### Fixes:
 
 * Put fixes here (in a brief bullet point)
+* Also exclude the `tmp`, `log` and `storage` directories from being inspected.
+  Working on a Rails application, especially the `tmp` directory fills up over time and slows down linting the project enormously. Same goes for the `storage` directory: here rubocop also has to dig through deeply nested folder structures.  
+  Note: rubocop's "default" configuration also ignores the `tmp` directory.
```

Comment:
👍 thanks for the extensive explanation

we should think to add a "add you feature/change to the changelog" line in our PR_TEMPLATE 🤔

Reply:
Yes, would make sense 👍 
One can easily forget and therefore making it harder for the person who's in charge of releasing the next version (as they have to dig out the information for the commit history first).

Reply:
Got us covered: https://github.com/bitcrowd/rubocop-bitcrowd/pull/20